cmd/chronicle/cli/cli.go

```diff
@@ -0,0 +1,49 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/anchore/chronicle/chronicle"
+	"github.com/anchore/chronicle/cmd/chronicle/cli/commands"
+	"github.com/anchore/chronicle/internal"
+	"github.com/anchore/chronicle/internal/version"
+	"github.com/anchore/clio"
+)
+
+func New() *cobra.Command {
+	versionInfo := version.FromBuild()
+	id := clio.Identification{
```

Comment:
the ID is a main package concern as well, which values should be explicitly passed. In this way anyone wanting to use the cli package can create their own identity.

https://github.com/anchore/quill/blob/17d942431c4b1537df0c4f6a02f8231b322d9d51/cmd/quill/main.go#L27-L33